### DISCUSSION TITLE
Error when using -oI without --remove-wildcard flag

### DISCUSSION THREAD

**Discussion by lushythedev** [projectdiscovery/subfinder#1126] [Q&A]
### Subfinder version:
Current subfinder version v2.6.4 (latest)

### Current Behavior:
`[FTL] Program exiting: hostip flag must be used with RemoveWildcard option`

### Expected Behavior:
I expected the command '`subfinder -d example.com -oI --remove-wildcard`' to successfully execute to include host IP addresses in the output with the. I also just tried `subfinder -d example.com -oI`, but said '`[FTL] Program exiting: hostip flag must be used with RemoveWildcard option`'

### Steps To Reproduce:
Example: steps to reproduce the behavior:
1. Run '`subfinder -d example.com -o output.txt -oI --remove-wildcard`' (assuming `--remove-wildcard` is the command?)
2. See the error message.

**Comment by ehsandeep** [comment]
@lushythedev as suggested in the help option

```
subfinder -h oI
Subfinder is a subdomain discovery tool that discovers subdomains for websites by using passive online sources.

Usage:
  subfinder [flags]

Flags:
   -oI, -ip  include host IP in output (-active only)
```

Valid run:

```
subfinder -d example.com -o output.txt -oI -active
```

**Reply by lushythedev** [reply]
Hi,

Doesn't seem to enumerate any domains with IP addresses:

Command ran `subfinder -d example.org -o output.txt -oI -active`

```
[INF] Current subfinder version v2.6.4 (latest)
[INF] Loading provider config from /Users/lushy/Library/Application Support/subfinder/provider-config.yaml
[INF] Enumerating subdomains for example.org
[INF] Found 0 subdomains for example.org in 1 minute 41 seconds
```

Command ran normally:

`subfinder -d example.org`

Works as expected:

```
[INF] Found 142 subdomains for example.org in 1 second 959 milliseconds
```
